src/charm.py

```diff
@@ -197,6 +227,34 @@ def _network_attachment_definitions_from_config(self) -> list[NetworkAttachmentD
             ),
         ]
 
+    def _request_user_plane_network_to_ip_router(self) -> None:
```

Comment:
Question for @Kayra1  after sprint!!
Context:
on a config change we will request a new network and then get the routing table to get the UPF network information, to create the route and write config file in the container.
However, requesting a new network will trigger a new routing table updated event , in which we will get the routing table from the event and get the UPF network information to create the route and write config file in the container. AGAIN.